lib/features/interaction-events/InteractionEvents.js

```diff
@@ -45,10 +49,20 @@ var LOW_PRIORITY = 500;
  */
 export default function InteractionEvents(eventBus, elementRegistry, styles) {
 
-  var HIT_STYLE = styles.cls('djs-hit', [ 'no-fill', 'no-border' ], {
-    stroke: 'white',
-    strokeWidth: 15
-  });
+  function createHitStyle(element) {
+
+    var traits = [ 'no-fill', 'no-border' ];
+
+    if (isFrameElement(element)) {
```

Comment:
Even this could be handled via CSS, if a frame had the `djs-shape.djs-frame` classes.